src/Concordium/Client/GRPC2.hs

```diff
@@ -843,12 +843,24 @@ instance FromProto Proto.ConsensusInfo where
         csGenesisIndex <- fromProto $ ci ^. ProtoFields.genesisIndex
         csCurrentEraGenesisBlock <- fromProto $ ci ^. ProtoFields.currentEraGenesisBlock
         csCurrentEraGenesisTime <- fmap timestampToUTCTime . fromProto $ ci ^. ProtoFields.currentEraGenesisTime
+        currentTimeoutDuration <- fromProtoMaybe $ ci ^. ProtoFields.maybe'currentTimeoutDuration
+        currentRound <- fromProtoMaybe $ ci ^. ProtoFields.maybe'currentRound
+        currentEpoch <- fromProtoMaybe $ ci ^. ProtoFields.maybe'currentEpoch
+        triggerBlockTime <- (fmap . fmap) timestampToUTCTime . fromProtoMaybe $ ci ^. ProtoFields.maybe'triggerBlockTime
+        let csConcordiumBFTStatus = case (currentTimeoutDuration, currentRound, currentEpoch, triggerBlockTime) of 
+                (Just cbftsCurrentTimeoutDuration, Just cbftsCurrentRound, Just cbftsCurrentEpoch, Just cbftsTriggerBlockTime) -> Just ConcordiumBFTStatus{..}
+                _ -> Nothing
```

Comment:
I suppose technically we could raise an exception here if only some are set. But I guess we should trust the node and this'll work fine.